vulnerable_people_form/integrations/persistence.py

```diff
@@ -0,0 +1,199 @@
+import boto3
+
+from flask import current_app
+
+
+def get_rds_data_client(app=current_app):
+    return boto3.client(
+        "rds-data", endpoint_url=app.config.get("LOCAL_AWS_ENDPOINT_URL")
+    )
+
+
+def get_rds_client(app=current_app):
+    return boto3.client("rds", endpoint_url=app.config.get("LOCAL_AWS_ENDPOINT_URL"))
+
+
+def get_secretsmanager_client(app=current_app):
+    return boto3.client(
+        "secretsmanager", endpoint_url=app.config.get("LOCAL_AWS_ENDPOINT_URL")
+    )
+
+
+def _find_database_arn(app):
+    rds = get_rds_client(app=app)
+    clusters = rds.describe_db_clusters()["DBClusters"]
+    coronavirus_rds = [
+        x
+        for x in clusters
+        if x["DBClusterIdentifier"].startswith(app.config["DATABASE_CLUSTER_PREFIX"])
+    ][0]
+    return coronavirus_rds["DBClusterArn"]
+
+
+def _find_database_secret_arn(app):
+    secrets_manager = get_secretsmanager_client(app=app)
+    return secrets_manager.list_secrets(
+        Filters=[{"Key": "tag-value", "Values": app.config["DATABASE_SECRET_TAGS"]}]
+    )["SecretList"][0]["ARN"]
+
+
+def init_app(app):
+    if app.config.get("AWS_RDS_DATABASE_ARN_OVERRIDE"):
+        app.config["AWS_RDS_DATABASE_ARN"] = app.config["AWS_RDS_DATABASE_ARN_OVERRIDE"]
+    else:
+        app.config["AWS_RDS_DATABASE_ARN"] = _find_database_arn(app)
+
+    if app.config.get("AWS_RDS_SECRET_ARN_OVERRIDE"):
+        app.config["AWS_RDS_SECRET_ARN"] = app.config["AWS_RDS_SECRET_ARN_OVERRIDE"]
+    else:
+        app.config["AWS_RDS_SECRET_ARN"] = _find_database_secret_arn(app)
+
```

Comment:
The purpose of all the `_find_..._arn()` complications is that the arn values are capable of changing without warning

Have you got a fallback anywhere to refresh these values if the job fails?